project-templates/import-records/format-people-config.js

```diff
@@ -0,0 +1,107 @@
+var normalize = require('medic-bulk-utils').normalize,
```

Comment:
I don't really get this. Why would you format people if you're importing records? Is this a template for something that can import records **and** people?